code/src/components/EmptyState.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+export const EmptyState = (props) => {
+  return (
+    <>
+      <div className="empty-state"><h1>{props.message}</h1></div>
+    </>
```

Comment:
If your component returns only one root element, there is no need to wrap it around React Fragment.

In this case WE NEED to use React Fragment :

```
return (
  <div>My element 1</div>
  <div>My element 2</div>
);
```

so, after necessary changes it looks like this :

```
return (
  <>
    <div>My element 1</div>
    <div>My element 2</div>
  </>
);
```

But in this case WE DON'T NEED to use React Fragment :

```
return (
  <div>My element 1</div>
);
```